server/checkdoc.js

```diff
@@ -0,0 +1,32 @@
+const Doc = require('doc');
+const Taboo = require('taboo');
+
+const inputdoc = (req, res) => {
+  let docinst = new Doc({
+    title: req.body.name,
+    content: req.body.text,
+    owner_id: req.body.owner_id,
+    locked: false,
+    revision_id: 0,
+    last_revised: null
+  });
+
+  Taboo.find({}, function (err, results) {
+    if (err) {
+      throw err;
+    }
+    else {
+      console.log(results);
+      var taboo_words_arr = results;
```

Comment:
This will not work how you expect it to since this is a callback.
Also, do not use var.

Reply:
rearranged variable